src/System.CodeDom/tests/CSharpCodeGenerationTests.cs

```diff
@@ -614,125 +615,160 @@ public void Arrays_WithinArrays_Empty_NonPrimitiveTypes()
-                            new CodeAttributeArgument("IsNullable", new CodePrimitiveExpression(false))));
-            method1.Parameters.Add(param2);
-            class1.Members.Add(method1);
-
-            var function1 = new CodeMemberMethod();
-            function1.Name = "MyFunction";
-            function1.ReturnType = new CodeTypeReference(typeof(string));
-            function1.CustomAttributes.Add(new CodeAttributeDeclaration("System.Obsolete", new CodeAttributeArgument(new CodePrimitiveExpression("Don't use this Function"))));
-            function1.ReturnTypeCustomAttributes.Add(new CodeAttributeDeclaration("System.Xml.Serialization.XmlIgnoreAttribute"));
-            function1.ReturnTypeCustomAttributes.Add(new CodeAttributeDeclaration("System.Xml.Serialization.XmlRootAttribute", new
-                CodeAttributeArgument("Namespace", new CodePrimitiveExpression("Namespace Value")), new
-                CodeAttributeArgument("ElementName", new CodePrimitiveExpression("Root, hehehe"))));
-            function1.Statements.Add(new CodeMethodReturnStatement(new CodePrimitiveExpression("Return")));
-            class1.Members.Add(function1);
-
-            CodeMemberMethod function2 = new CodeMemberMethod();
-            function2.Name = "GlobalKeywordFunction";
-            function2.CustomAttributes.Add(new CodeAttributeDeclaration(new CodeTypeReference(typeof(ObsoleteAttribute), CodeTypeReferenceOptions.GlobalReference), new
-                CodeAttributeArgument(new CodePrimitiveExpression("Don't use this Function"))));
-            CodeTypeReference typeRef = new CodeTypeReference("System.Xml.Serialization.XmlIgnoreAttribute", CodeTypeReferenceOptions.GlobalReference);
-            CodeAttributeDeclaration codeAttrib = new CodeAttributeDeclaration(typeRef);
-            function2.ReturnTypeCustomAttributes.Add(codeAttrib);
-            class1.Members.Add(function2);
-
-            CodeMemberField field1 = new CodeMemberField();
-            field1.Name = "myField";
-            field1.Type = new CodeTypeReference(typeof(string));
-            field1.CustomAttributes.Add(new CodeAttributeDeclaration("System.Xml.Serialization.XmlElementAttribute"));
-            field1.InitExpression = new CodePrimitiveExpression("hi!");
-            class1.Members.Add(field1);
+                var attrs = cu.AssemblyCustomAttributes;
+                attrs.Add(new CodeAttributeDeclaration("System.Reflection.AssemblyTitle",
+                    new CodeAttributeArgument(new CodePrimitiveExpression("MyAssembly"))));
+                attrs.Add(new CodeAttributeDeclaration("System.Reflection.AssemblyVersion",
+                    new CodeAttributeArgument(new CodePrimitiveExpression("1.0.6.2"))));
+                attrs.Add(new CodeAttributeDeclaration("System.CLSCompliantAttribute",
+                    new CodeAttributeArgument(new CodePrimitiveExpression(false))));
```

Comment:
Can you please revert all of this newline / whitespace formatting that was done?

Reply:
I will.